modules/cactus-web/src/DateInput/DateInput.test.tsx

```diff
@@ -320,20 +320,87 @@ describe('component: DateInput', (): void => {
       const desiredDate = PartialDate.from('2018-03-23', { type: 'date', format: 'YYYY-MM-dd' })
       await act(
         async (): Promise<void> => {
-          fireEvent.click(getByLabelText('Click to change month and year'))
+          fireEvent.click(getByLabelText('Click to change month'))
           await animationRender()
           fireEvent.click(getByText('March'))
+          fireEvent.click(getByLabelText('Click to change year'))
+          await animationRender()
           fireEvent.click(getByText('2018'))
-          fireEvent.click(getByLabelText('Click to use calendar picker'))
           await animationRender()
           // @ts-ignore
           fireEvent.click(getByText(desiredDate.toLocaleSpoken('date')).parentElement)
         }
       )
-      expect(handleChange).toHaveBeenCalledTimes(1)
+      expect(handleChange).toHaveBeenCalledTimes(3)
       expect(value).toEqual('2018-03-23')
     })
 
+    test('can change month using left arrow', async (): Promise<void> => {
+      let value: any
+      const handleChange = jest.fn((e) => {
+        value = e.target.value
+      })
+      const { getByLabelText } = render(
+        <StyleProvider>
+          <DateInput
+            name="date_input"
+            id="date-input"
+            format="YYYY-MM-dd"
+            value="2018-03-23"
+            onChange={handleChange}
+          />
+        </StyleProvider>
+      )
+
+      const portalTrigger = getByLabelText('Open date picker')
+      act((): void => {
+        fireEvent.mouseDown(portalTrigger)
+        fireEvent.click(portalTrigger)
+      })
```

Comment:
Just a thought, but I wonder if this block can be replaced with the following:

```suggestion
      userEvent.click(portalTrigger)
```